core/io/resource_loader.h

```diff
@@ -226,7 +226,7 @@ class ResourceLoader {
 	// Loaders can safely use this regardless which thread they are running on.
 	static void notify_load_error(const String &p_err) {
 		if (err_notify) {
-			callable_mp_static(err_notify).bind(p_err).call_deferred();
+			MessageQueue::get_main_singleton()->push_callable(callable_mp_static(err_notify).bind(p_err));
```

Comment:
```suggestion
			MessageQueue::get_main_singleton()->push_callable(callable_mp_static(err_notify), p_err);
```
Unnecessary, see the signature of the method